### ISSUE TITLE
Required Diesel features are not documented

### ISSUE THREAD

**Issue by shepmaster** [quodlibetor/diesel-derive-newtype#29]
Create a project:

```
cargo new repro

cd repro

cargo add diesel diesel-derive-newtype --no-default-features --features=diesel/postgres
```

Use the derives: 

```rust
use diesel_derive_newtype::DieselNewType;

#[derive(Debug, DieselNewType)]
struct Wrapper(i64);

fn main() {}
```

Build it:

```
cargo build

error[E0412]: cannot find type `RawValue` in module `diesel::backend`
 --> src/main.rs:3:17
  |
3 | #[derive(Debug, DieselNewType)]
  |                 ^^^^^^^^^^^^^ not found in `diesel::backend`
  |
  = note: this error originates in the derive macro `DieselNewType` (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by shepmaster** [comment]
Ideally, this crate would work with the default features disabled.

**Comment by shepmaster** [comment]
It appears that the feature in question is `with-deprecated`.

**Comment by shepmaster** [comment]
Ah, this may be fixed by #28.

**Comment by quodlibetor** [comment]
This should be resolved in crates.io v2.1.0, feel free to re-open if it's still broken.
